### PR TITLE
TIP-706-categories: Add tests

### PR DESCRIPTION
**Pull request by anaelChardan**
**ES Filter Checklist:**
Update tests and add Normalisation

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | OK
| Added Behats                      | -
| Added integration tests           | OK
| Changelog updated                 | -
| Review and 2 GTM                  | OK
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Component/Catalog/spec/Normalizer/Indexing/Product/PropertiesNormalizerSpec.php [hunk 1]
```diff
@@ -69,6 +71,13 @@ function it_normalizes_product_fields_and_values(
             ->willReturn($productValueCollection);
         $productValueCollection->isEmpty()->willReturn(false);
 
+        $product->getCategoryCodes()->willReturn(
```
Comment (damien-carcel) [inline src/Pim/Component/Catalog/spec/Normalizer/Indexing/Product/PropertiesNormalizerSpec.php:76]: why is this method commented? If there is a reason, you should add a comment at the beginning.

**Review by damien-carcel — APPROVED** [review]
One question, but GTM.

File: src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst [hunk 2]
```diff
@@ -753,7 +753,8 @@ Example with the ``>`` operator:
 
 Category
 ********
-:Apply: apply on 'categories' field
+:Apply: 'keyword' datatype
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst]: can you use the same semantic that Samir's?
https://github.com/jjanvier/pim-community-dev/pull/75/files#diff-f46b60971d1e2bc2731d390e2d33050dR905

**Review by jjanvier — COMMENTED** [review]
GTM if integration tests are green

**Review by jjanvier — APPROVED** [review]
GTM if integration tests are green

**Review by SamirBoulil — APPROVED** [review]
GTM same question as Damien

**Comment by SamirBoulil** [comment]
Build is here:
https://core-ci.akeneo.com/blue/organizations/jenkins/jjanvier%2Fpim-community-dev/detail/PR-68/11/tests

Category PQB filter tests is green.
Specs are green

**Pull request merged**